content/02.introduction.md

```diff
@@ -26,9 +26,9 @@ However, direct comparison among different data types for this application is la
 
 We select a collection of potential cancer drivers with varying functions and roles in cancer development [@doi:10.1126/science.1235122].
 We use mutation status in these genes as labels to train classifiers, using each of the data types listed as training data, in a pan-cancer setting; we follow similar methods to the elastic net logistic regression approach described in Way et al. 2018 [@doi:10.1016/j.celrep.2018.03.046] and Way et al. 2020 [@doi:10.1186/s13059-020-02021-3].
-We show that there is considerable predictive signal for many genes relative to a random baseline, and that gene expression and DNA methylation generally provide the best predictions of mutation state.
+<!--From here down in this paragraph feels a bit abstract-y to me, since I'm not sued to seeing a summary of results in the intro. Typically I would expect to see one more sentence here just bringing it back big-picture, and then to find the following sentences in the discussion -->
```

Comment:
```suggestion
```
(will address in future PR)

Reply:
After looking through some published papers, it seems journal-dependent but I do think it's somewhat common to have a sentence or two summarizing the main findings at the end of the intro. I'll trim this summary down a bit but I think I'm going to leave one or two sentences touching on our main findings - if we end up needing to remove them once the paper is accepted, we can of course do that.

Reply:
That definitely makes sense! I was wondering if it was a field difference (since I've published in this area!)